.nvmrc

```diff
@@ -1 +1 @@
-14
\ No newline at end of file
+18
```

Comment:
16 or 18?

Reply:
18 - this is for our local dev, and what engines specifies is the version of node that our app will use

so overall we'll be using node 18 for local dev, and node 16 for deployment - this is fine since we're not using any node 18 specific features

Reply:
makes sense, thanks!